client/verta/tests/conftest.py

```diff
@@ -277,6 +289,17 @@ def client(host, port, email, dev_key):
     print("[TEST LOG] test teardown completed {} UTC".format(datetime.datetime.utcnow()))
 
 
+@pytest.fixture
+def client_2(host, port, email_2, dev_key_2):
+    # For collaboration tests
```

Comment:
Since it describes the whole function, it should be a docstring rather than a comment.
```suggestion
    """For collaboration tests."""
```

Reply:
Fixed.